src/analytics/README.md

```diff
@@ -1,32 +1,60 @@
 fetch branch from github.
```

Comment:
Maybe we could focus on setting up the default branch (master) and add instructions to change to the analytics micro-service directory.
example
```suggestion
clone project from github.
- `git clone https://github.com/airqo-platform/AirQo-api.git`

change directory to the analytics micro-service
- `cd src/analytics`
```

Reply:
Thanks, that's better!